hibernate-envers/src/test/java/org/hibernate/envers/test/integration/onetomany/BasicSetWithMulId.java

```diff
@@ -94,11 +94,9 @@ public void testRevisionsCounts() {
 	}
 
 	private <T> Set<T> makeSet(T... objects) {
-		Set<T> ret = new HashSet<T>();
+		Set<T> ret = new HashSet<T>(objects.length);
```

Comment:
Again, this size spec is usually not what user wants. See https://guava.dev/releases/23.0/api/docs/src-html/com/google/common/collect/Sets.html#line.252

Reply:
Same as above, explicitly setting the loadfactor using the two parameter constructor would make sure that you explicitly create a hashset of exactly the needed size.